src/components/Content.tsx

```diff
@@ -17,55 +17,109 @@ const Content = () => {
   const [isHeightLargerThan1000] = useMediaQuery('(min-height: 1000px)')
   // const flexDirection=isLessThan1250 ?"column":"row";
   // console.log(isLessThan1250,"less")
+  const [showText1, setShowText1] = useState(true);
+
+  // Array of texts and links
+  const texts = [
+    {
+      text: 'Airdrop for mainnet v1 ',
+      link: 'https://app.hashstack.finance/',
+    },
+    {
+      text: 'Referral Campaign is live ',
```

Comment:
Comment this one out.

Reply:
@v9hstk This one is just kept for future reference if we need to use it, basically this is an array of texts and links we only use the upper one for now